src/OSPSuite.Core/Domain/ObjectPath.cs

```diff
@@ -163,13 +163,23 @@ public void Remove(string entry)
          if (_pathEntries.Count == 0)
             return null;
 
-         var firstEntry = _pathEntries[0];
-         var root = refEntity.RootContainer;
          var usePath = new List<string>(_pathEntries);
-         IEntity dependentObject;
+
+         var dependentObject = refEntity;
+         //Do we have a valid relative path from the current object
```

Comment:
This solves the issue with the Mean parameter. In this case, we have to check the relative path first and then if nothing comes out, try absolute.